Cannot build new docker image

<!-- NOTE: This template is NOT a suggestion. Issues not using this template will be marked invalid. -->

### Description
Cannot create OpenLane docker image with updated OR version.

### Environment
```
Python: v3.8.8
Kernel: Darwin v21.2.0
Distribution: macOS 10.16
Container Engine: docker v20.10.11
OpenLane Git Version: 2021.10.08_02.00.18
---
PDK Version Verification Status: FAILED
/Users/vvbandeira/LocalFiles/openroad-local/openlane/pdks/sky130A not found.
Traceback (most recent call last):
  File "/Users/vvbandeira/LocalFiles/openroad-local/openlane/dependencies/verify_versions.py", line 124, in verify_versions
    raise Exception(f"{sky130_dir} not found.")
Exception: /Users/vvbandeira/LocalFiles/openroad-local/openlane/pdks/sky130A not found.

Failed to compare PDKs.
---
Git Log (Last 3 Commits)

commit 80c55b0c253359543a03193368d6e81be6430100
Author: Donn <me@donn.website>
Date:   Tue Dec 28 17:21:57 2021 +0200

    Update OpenROAD SWIG to 4.0.1 (#814)

    + Add llvm-devtoolset-7-clang to CentOS 7 dependencies, just in case (OpenROAD was being funny with GCC for a second there)
    ~ Updates OpenROAD SWIG to 4.0.1
    ~ Update scripts to support changes in newer versions of OpenROAD
    ~ Onus of building images moved to the OpenLane CI proper instead of the tool update flow, which now just basically edits one file

commit 9e61eee9524034c0a930d38a643823045993e17a
Author: FaragElsayed2 <91015308+FaragElsayed2@users.noreply.github.com>
Date:   Tue Dec 28 13:08:26 2021 +0200

    Update run_designs.py Documentation (#812)

    + documented --threads option
    - removed documentation for --designs option for run_designs.py

    [skip ci]

commit 4aff44eb405654e93ea87feca0b3201b831d471e
Author: Donn <me@donn.website>
Date:   Mon Dec 27 14:34:20 2021 +0200

    Consolidate DEF Manipulation Scripts (#807)

    + Created defutil.py with functionality of all removed scripts (save remove_empty_pins and remove_empty_ports,) reimplemented properly using odb instead of regular expressions
    ~ Folded consoletext.py into the padframe generator
    - Removed li1 hack commands, using the RT_MAX_LAYER/RT_MIN_LAYER variables should be sufficient now that it's passed on to DRT
    - Removed a huge number of shell and python regex-based DEF manipulation scripts
```

### Reproduction Material
```
make -C docker openlane
```

### Expected behavior
Create OL docker image.

### Logs
```
make -C docker openlane
rm -f tar/cugr.tar.gz && echo "Deleted existing tar/cugr.tar.gz"
Deleted existing tar/cugr.tar.gz
mkdir -p tar
python3 ./utils.py pull-if-doesnt-exist --repository efabless/openlane-tools --os centos-7 cugr
Usage: utils.py pull-if-doesnt-exist [OPTIONS] TOOL
Try 'utils.py pull-if-doesnt-exist --help' for help.

Error: Invalid value for '-o' / '--os': 'centos-7' is not one of 'c', 'e', 'n', 't', 'o', 's', '-', '7'.
make: *** [tar/cugr.tar.gz] Error 2
```

I think the argument for click choice [here](https://github.com/The-OpenROAD-Project/OpenLane/blob/80c55b0c253359543a03193368d6e81be6430100/docker/utils.py#L31) should be a list, not a string.

--- a/docker/utils.py
+++ b/docker/utils.py
@@ -28,7 +28,7 @@ def cli():
 
 @click.command("pull-if-doesnt-exist")
 @click.option("-r", "--repository", required=True)
-@click.option("-o", "--os", "operating_system", required=True, type=click.Choice('centos-7'))
+@click.option("-o", "--os", "operating_system", required=True, type=click.Choice(['centos-7']))
 @click.argument("tool")
 def pull_if_doesnt_exist(repository, operating_system, tool):
     image_tag = subprocess.check_output([

